apps/api/src/routes/v1_ratelimit_listOverrides.ts

```diff
@@ -0,0 +1,121 @@
+import { rootKeyAuth } from "@/pkg/auth/root_key";
+import { openApiErrorResponses } from "@/pkg/errors";
+import type { App } from "@/pkg/hono/app";
+import { createRoute, z } from "@hono/zod-openapi";
+import { buildUnkeyQuery } from "@unkey/rbac";
+
+const route = createRoute({
+  tags: ["ratelimit"],
+  operationId: "listOverrides",
+  method: "get",
+  path: "/v1/ratelimit.listOverrides",
+  security: [{ bearerAuth: [] }],
+  request: {
+    query: z.object({
+      namespaceId: z.string().openapi({
+        description:
+          "Namespaces group different limits together for better analytics. You might have a namespace for your public API and one for internal tRPC routes.",
+        example: "email.outbound",
+      }),
+      identifier: z.string().optional().openapi({
```

Comment:
what does this field do?

Reply:
think of it from the perspective of a user when they read this, and then explain to them what it does.